core-backend/wasmtime/src/funcs_tree_builder.rs

```diff
@@ -0,0 +1,61 @@
+use crate::{env::StoreData, funcs::FuncsHandler};
```

Comment:
But in general, I don't see any reason to separate this logic into separate logic for both backends